### PR TITLE
Make JAVA SDK camelCased meeting-attendee response compatible with JS SDK

### PR DESCRIPTION
**Pull request by RidipDe**
**Issue #:**
Java SDK doesn't return responses as JSON. It is returned as structured Java objects as mentioned in the public documentation https://sdk.amazonaws.com/java/api/latest/software/amazon/awssdk/services/chime/model/CreateMeetingResponse.html

The Chime service sends a JSON response with upper-case starting keys as per the API documentation.
The Java SDK is then deserializing that into a structured response object which might have method names that start with lower-case characters. 

**Description of changes:**
With this PR we are making the properties/keys of the CreateMeeting and CreateAttendee response case insensitive by converting the keys to lower case 

**Testing**

1. Have you successfully run `npm run build:release` locally?
Yes

2. How did you test these changes?
Deploying and running the serverless app - was successfully able to successfully join a meeting

3. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved?
Yes, discussed

4. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved?
No

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/meetingsession/MeetingSessionConfiguration.ts [hunk 1]
```diff
@@ -172,4 +180,34 @@ export default class MeetingSessionConfiguration {
       this.credentials ? this.credentials.attendeeId : null
     );
   }
+
+  private toLowerCasePropertyNames(jsonObject: any) { // eslint-disable-line
```
Comment (simmkyu) [inline src/meetingsession/MeetingSessionConfiguration.ts:184]: Two conversions-from an object to Map to JSON-are not required. Use this function that also handles a non-object and an array type (e.g. an object in an array).

```
// eslint-disable-next-line @typescript-eslint/no-explicit-any
private toLowerCasePropertyNames(input: any): any {
  if (typeof input !== 'object') {
    return input;
  } else if (Array.isArray(input)) {
    return input.map(toLowerCasePropertyNames);
  };
  // eslint-disable-next-line @typescript-eslint/no-explicit-any
  return Object.keys(input).reduce((result: any, key: string) => {
    const value = input[key];
    const newValue = typeof value === 'object' ? toLowerCasePropertyNames(value) : value;
    result[key.toLowerCase()] = newValue;
    return result;
  }, {});
};
```

File: src/meetingsession/MeetingSessionConfiguration.ts [hunk 2]
```diff
@@ -138,27 +138,35 @@ export default class MeetingSessionConfiguration {
    * ```
    */
   constructor(createMeetingResponse?: any, createAttendeeResponse?: any) { // eslint-disable-line
+    if (createAttendeeResponse) {
```
Comment (simmkyu) [inline src/meetingsession/MeetingSessionConfiguration.ts:141]: nit: Convert the attendee response within one `if` below, similar to the meeting response.

File: test/meetingsession/MeetingSessionConfiguration.test.ts [hunk 3]
```diff
@@ -79,6 +79,68 @@ describe('MeetingSessionConfiguration', () => {
       expect(configuration.credentials.joinToken).to.eq('join-token');
     });
 
+    it('can take a CreateMeeting and CreateAttendee response object using JAVA SDK', () => {
+      const configuration = new MeetingSessionConfiguration(
+        {
+          meetingid: 'meeting-id',
```
Comment (simmkyu) [inline test/meetingsession/MeetingSessionConfiguration.test.ts:85]: AWS SDK for Java uses `camelCase`, e.g. `audioHostURL`

**Review by simmkyu — CHANGES_REQUESTED** [review]

File: src/meetingsession/MeetingSessionConfiguration.ts (hunk 1 above)
Comment (simmkyu) [reply src/meetingsession/MeetingSessionConfiguration.ts:184]: 1. [Click here to see the TypeScript playground](https://www.typescriptlang.org/play?#code/PTAEFMGcBsEsDsAuBaAJrSBDARtcz5wAPFOQ0AAUQE8AHKAYwCdZaUozFh4B7ZY2nAawUmeNQBQAMwCu8Bolg94oRDwAyPAO7gmAYUyRwABSY96TGgDlMAWygAKBLRmIAXKDHUAlB6+gAbwlQENBYKVAHGnoeCOdXUABCAF5k0AByHmwAK3AFdO9A4NCSpnBEGSYVeMQAbmKQgF8IaCMwiIcAQSYmTGoAOgxu3uoneBdEb0KgktLyyurx137bTFoojW1dAyNTc11rOyhvepLG09CQCBgEFHQsXHxCEmQycEpoxhY2fhukbj4AiEImQXgaoDKFSqoAA8jk8oh+gBrcDUSBjCbefplVAyBjgBwOMqQGTQdyecQAGlAKOoHkgiBY8AA5oVkgA+IqzUIMZQM0AAN0w0Bk7zSNQA2rSALoXbm8+D8whaABqwtFoDSn1igvVYtSGSyuXyoAA-KpNjp9IYTGYLId7OihSLwIUPM7RXLZsTSYgpaj+mpNFadgTvNLNaBlWqXV65lCVD6yV7GtSAo0ThJzhIJAr+TVIzNQAAiABSPAQAEl4FIeMWPEWQsWALLgcoIZn1rmzFttxQsruNnut9ssyuoLvFgAcU4AzAB2BizgBsAAZkABOVDYKfIAAsAFZ5wBGTeYPez-izg8HqSoKcAJlQmFQy+LlPBJWLAFESLp4MKI79sy46TgAyogYioM+0BSEi2QTh+3JNq26CYMY0CYPi9hIIOn49p0MjoDwAASPAMgAqkw0CTg+x7gMeqDgFI85SMee5bqu4BPpgmDHses6zqgB7XqgDBSI+yIrA+-QyFox79GstD9AwAAWsD2IpWiQG4s57vOU7vvhX6EcRABiwrQNgWFIlRNEeMW2k6SAqmYEQTDKbYMlyQpSkqepmmYNpbh7hewAMJZkDADOC5Lmum7bruh4nmeF5Xjed6Ps+r5GchoTFmBzBtvAAAimCQXZk5OW4IDYCItBaNJsnyYptDKWpGngFpOmhbOwACg+wCQEV4DwNFc6Liu65bju+5HqeG7npe4DXre948TlSF5SWhVlKNYGuUyzKVQ51W1fVjVec1vltf5nXdSFYUDUNI1jTFk3xTNSXzaly2rZlG1vlteUFa9KqwOAWgdidJZncAdWIA1TU+a17UBV1QU9WF2nhcohAKKatCGMNPBMck8CktAABkAoQ1aAD6MgyLAqDk5TVMABrIAAQiIJEyNgyAGNA0DIOOyTvXF02JXNKWLWlK0Zet2VA8Z+VgbAzIAWQx3UVVkDOUNmva8jLV+R1gU4wqjI8NA42xVNCWzclC1Lela1ZS+qvbSWAAqCx6MoNvQDDxaqYgiOGwwDDtcpoo3Wj92Y-1g0JvTRgG0oirFmrqZq72aEAErgMyWeTjIkDIDoDLIMeOfco04J5yZEejUx4B4dyxadK38Dt53yE-n+VTChRRhMKBDl7tgx44POO4AMTPlI0BIoYqDZLlg894gbdtpPJasQwG7T1O66rs+l6zuAHFoMux5SLLmAPlO4B8YtmBb135YIL7PAovAScVgHwAC1sgwl9oXWAMISo92bAAcULqpZs1BVwAC8QG+wAIoHnUAAdSwQ+ZseCACaiArAlQorOEhvtKx7hAXg38MI8FmVUjCbI34iG+2ZA+KwvsKKrnUL7EBsASHZCweQ+BKpsggLAseVSNDWFCI3ggiiRAYTwIoloTBlZZzNiwfXWYjczjFEbtmXMfJbZdWgDwZkGxgzbBtHse01AbCOgxK4KYtQgA)
2. Open the browser console and click on "Run." (Or view the Logs tab in the right panel.)

**Comment by RidipDe** [comment]
Closing PR for now, will open after the next push

File: test/meetingsession/MeetingSessionConfiguration.test.ts [hunk 4]
```diff
@@ -79,6 +79,128 @@ describe('MeetingSessionConfiguration', () => {
...
+        {
+          meetingId: 'meeting-id',
+          mediaPlacement: {
+            audioHostUrl: 'audio-host-url',
+            screenDataUrl: 'screen-data-url',
+            screenSharingUrl: 'screen-sharing-url',
+            screenViewingUrl: 'screen-viewing-url',
+            signalingUrl: 'signaling-url',
+            turnControlUrl: 'turn-control-url',
+            region: { regionPrimary: 'region-1', regionSecondary: ['region-2', 'region-3'] },
```
Comment (simmkyu) [inline test/meetingsession/MeetingSessionConfiguration.test.ts:185]: You can add a comment here that including an array is only for testing.

**Review by simmkyu — DISMISSED** [review]

**Review by simmkyu — APPROVED** [review]

**Review by simmkyu — DISMISSED** [review]

**Review by simmkyu — APPROVED** [review]

**Pull request merged**